### PR TITLE
Sass fixes

### PR DESCRIPTION
**Pull request by vegeorgi**
1. Fix sass with webpack.
2. Add platform specific files for drawer component
3. Add page central icon based on the prototype
4. tns-android is updated to 3.0.1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
       "version": "3.0.0"
     },
     "tns-android": {
-      "version": "3.0.0"
+      "version": "3.0.1"
```
Comment (ADjenkov) [inline package.json]: Suggestion: I think we should mention that in the PR description when such a __big__ change occurs

File: app/shared/drawer.component.html [hunk 2]
```diff
@@ -8,7 +8,7 @@
     <StackLayout class="sidedrawer-content">
         <ListView [items]="items" separatorColor="transparent">
             <ng-template let-item="item" let-i="index" let-odd="odd" let-even="even">
-                <GridLayout rows="*" columns="auto, *"
+                <GridLayout rows="*" columns="auto, *" rows="50"
```
Comment (ADjenkov) [inline app/shared/drawer.component.html]: No need to set `rows` when only one row will be presented. By default the defult row in the grid will be __*__ stretch. 

What is this `rows="50"` ?? Typo? I don't think we need any `rows` definition here especially 2 definitions

File: app/browse/browse.component.html [hunk 3]
```diff
@@ -12,7 +12,11 @@
         <MyDrawer [selectedPage]="'Browse'"></MyDrawer>
     </StackLayout>
     
-    <StackLayout tkMainContent>
-        <Label text="Browse Page"></Label>
+    <StackLayout class="page" tkMainContent>
+        <AbsoluteLayout class="pageIcon">
```
Comment (ADjenkov) [inline app/browse/browse.component.html]: What is the case with the `AbsoluteLayout` ? No other way to position the icons may be?

File: package.json (hunk 1 above)
Comment (vegeorgi) [inline package.json]: 👍

File: app/shared/drawer.component.html (hunk 2 above)
Comment (vegeorgi) [inline app/shared/drawer.component.html]: 👍 
yep, I`ve missed to remove the first one - rows="*".
The rows="50" fix the problem with the drawer items height on initial load([the bug](https://github.com/NativeScript/NativeScript/issues/4294#event-1105784223)). This is a temporary fix, which if you do not want, I`m ok to be reverted.

File: app/browse/browse.component.html (hunk 3 above)
Comment (vegeorgi) [inline app/browse/browse.component.html]: This icons has to be as a background image for the page without messing up other page content. I will think about another solution, but for now, this is it

Comment (vegeorgi) [inline app/browse/browse.component.html]: I will make some changes here :)

**Comment by manoldonev** [comment]
@vegeorgi Do we really need to keep the *.css files that have matching *scss in source control?

File: search/search.component.html [hunk 4]
```diff
@@ -1,4 +1,6 @@
 <ActionBar class="action-bar">
+    <!--HACK - we should remove the navigation button-->
+    <NavigationButton visibility="collapsed"></NavigationButton>
     <ActionItem (tap)="openDrawer()"
```
Comment (manoldonev) [inline search/search.component.html]: Getting `JS: ERROR TypeError: co.openDrawer is not a function` here -- can you update the tap handler to `(tap)="onDrawerButtonTap()" as it seems we have missed this previously.

Comment (vegeorgi) [inline search/search.component.html]: 👍  done

**Pull request merged**